src/display.rs

```diff
@@ -126,13 +110,40 @@ impl<'a> Display for TableFormatState<'a> {
     }
 }
 
+fn visit_table(
+    f: &mut Formatter,
+    table: &Table,
+    path: &Vec<&str>,
+    is_array_of_tables: bool,
+) -> Result {
+    if path.len() == 0 {
```

Comment:
Hm, I don't know why clippy on travis didn't catch this
```suggestion
    if path.is_empty() {
```